Library/Formula/bash.rb

```diff
@@ -6,16 +6,69 @@ class Bash < Formula
 
   stable do
     url "http://ftpmirror.gnu.org/bash/bash-4.3.tar.gz"
-    mirror "https://ftp.gnu.org/gnu/bash/bash-4.3.tar.gz"
```

Comment:
That'd be fine, although for long-term plans' sake both mirrors should be `https://`